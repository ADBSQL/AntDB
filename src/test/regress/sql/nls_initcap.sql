set grammar to oracle;
select nls_initcap('ab cde') from dual;
select nls_initcap('AAb  CDE') from dual;
select nls_initcap('\?!@#$%^&*()sa') from dual;
select nls_initcap('AA\?!@#$%^&*()AA') from dual;
select nls_initcap('\?!@#ss$%^&*()sC') from dual;
select nls_initcap('test你test好test') from dual;
select nls_initcap('1234SOU') from dual;
select nls_initcap(to_char('suv')) from dual;
select nls_initcap(to_number('NaN')) from dual;
