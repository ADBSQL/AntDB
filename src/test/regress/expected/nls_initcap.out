set grammar to oracle;
select nls_initcap('ab cde') from dual;
 NLS_INITCAP('AB CDE') 
-----------------------
 Ab Cde
(1 row)

select nls_initcap('AAb  CDE') from dual;
 NLS_INITCAP('AAB  CDE') 
-------------------------
 Aab  Cde
(1 row)

select nls_initcap('\?!@#$%^&*()sa') from dual;
 NLS_INITCAP('\?!@#$%^&*()SA') 
-------------------------------
 \?!@#$%^&*()Sa
(1 row)

select nls_initcap('AA\?!@#$%^&*()AA') from dual;
 NLS_INITCAP('AA\?!@#$%^&*()AA') 
---------------------------------
 Aa\?!@#$%^&*()Aa
(1 row)

select nls_initcap('\?!@#ss$%^&*()sC') from dual;
 NLS_INITCAP('\?!@#SS$%^&*()SC') 
---------------------------------
 \?!@#Ss$%^&*()Sc
(1 row)

select nls_initcap('test你test好test') from dual;
 NLS_INITCAP('TEST你TEST好TEST') 
---------------------------------
 Test你Test好Test
(1 row)

select nls_initcap('1234SOU') from dual;
 NLS_INITCAP('1234SOU') 
------------------------
 1234sou
(1 row)

select nls_initcap(to_char('suv')) from dual;
 NLS_INITCAP(TO_CHAR('SUV')) 
-----------------------------
 Suv
(1 row)

select nls_initcap(to_number('NaN')) from dual;
 NLS_INITCAP(TO_NUMBER('NAN')) 
-------------------------------
 Nan
(1 row)

